admin/assets/javascripts/gds-pupil-form.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Pupil access arrangements selection
+ */
+
+if (!window.MTCAdmin) {
+  window.MTCAdmin = {}
+}
+
+/* global $ */
+(function () {
+  'use strict'
```

Comment:
intentionally not at top of file?

Reply:
@GuyHarwood nope was a mistake, it's now moved on top for all the files I have introduced.